src/SonarQube.Client.Tests/Api/SonarQubeService_Lifecycle.cs

```diff
@@ -23,12 +23,14 @@
 using System.Threading.Tasks;
 using FluentAssertions;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using Moq.Protected;
 using SonarQube.Client.Helpers;
 
 namespace SonarQube.Client.Tests.Api
 {
     [TestClass]
-    public class SonarQubeService_ConnectAsync : SonarQubeService_TestBase
+    public class SonarQubeService_Lifecycle : SonarQubeService_TestBase
```

Comment:
The tests for connecting, disconnecting and disposing are all in this class, so I renamed it (didn't seem worth creating a separate test class for testing _Dispose_ with only a single test in it).